core/shared/src/main/scala/org/http4s/util/Renderable.scala

```diff
@@ -51,7 +50,7 @@ object Renderer {
     private val dateFormat =
       DateTimeFormatter
         .ofPattern("EEE, dd MMM yyyy HH:mm:ss zzz")
-        .withLocale(Locale.US)
+        // .withLocale(Locale.US) // Causes problems with Scala.js
```

Comment:
What problems do you get?

Reply:
Ahh, I can fix this one now. While crossing Skunk I learned that the US locale is not included with Scala.js by default and it falls back to another one. To fix I can add a depenency to https://github.com/gemini-hlsw/gemini-locales/, do you mind looking at my PR there to cross-build it for 2.12 and 3?